app/Livewire/Project/Service/FileStorage.php

```
@@ -56,6 +60,18 @@ public function mount()
         }
 
         $this->isReadOnly = $this->fileStorage->isReadOnlyVolume();
+        $this->syncData(false);
+    }
+
+    private function syncData(bool $toModel = false): void
```

PHPMD (2.15.0)

The method syncData has a boolean flag argument $toModel, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)